samples/bluetooth/direct_test_mode/README.rst

```diff
@@ -164,12 +164,14 @@ If you want to view the debug messages, follow the procedure in :ref:`testing_rt
 Requirements
 ************
 
-* The following development kit:
+The sample supports the following development kit:
 
 .. table-from-rows:: /includes/sample_board_rows.txt
    :header: heading
    :rows: nrf5340pdk_nrf5340_cpunet
 
+Additionally, the sample has the following requirements:
+
 * The Network core
```

Comment:
Unrelated to this PR, but if any of the reviewers can explain what we mean when we require "the network core", this would be good to clarify. ;)
Maybe that the network core must be programmed with the empty firmware sample?

Reply:
Its the other way around. This sample is meant to be run on the Network Core with empty firmware on the Application Core. However, a comment, mentioning this, is available in the build section of this file.

It is safe to remove this line. Previously, when listing sample compatible targets, we used DK names with links to UGs. I have probably added the Network Core part here to be more precise which core to use with nRF5340 DK. I think it is leftover from past changes.